app/views/admin/shared/_featured_link_fields.html.erb

```diff
@@ -15,8 +15,13 @@
 
   <%= form.fields_for(:featured_links, featured_links) do |featured_links_form| %>
     <div class="js-duplicate-fields-set well">
-      <%= featured_links_form.text_field :title, class: 'input-md-4' %>
-      <%= featured_links_form.text_field :url, label_text: 'URL' %>
+      <% if @translation_locale && @translation_locale.code != I18n.default_locale %>
```

Comment:
This conditional feels like it could do with a view test.

Reply:
I've added a feature test for adding translations to organisations, which covers the conditional in this partial.